packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-12/env_setup.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+
+# agoric-upgrade-12 specific env here...
+export USER2ADDR=$($binary keys show user2 -a --keyring-backend="test" 2> /dev/null)
+
+printKeys() {
+  echo "========== GOVERNANCE KEYS =========="
+  echo "gov1: $GOV1ADDR"
+  cat ~/.agoric/gov1.key || true
+  echo "gov2: $GOV2ADDR"
+  cat ~/.agoric/gov2.key || true
+  echo "gov3: $GOV3ADDR"
+  cat ~/.agoric/gov3.key || true
+  echo "validator: $VALIDATORADDR"
+  cat ~/.agoric/validator.key || true
+  echo "user1: $USER1ADDR"
+  cat ~/.agoric/user1.key || true
+  echo "user2: $USER2ADDR"
+  cat ~/.agoric/user2.key || true
+  echo "========== GOVERNANCE KEYS =========="
+}
+
+pushPrice () {
```

Comment:
note for @iomekam :

I think we have `agops oracle setPrice --keys=gov1,gov2` in this context. It obviates the weird stuff this code does.